src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="/big-search"
+      method="POST"
+      data-qa="big"
+    >
+      <img
+        class="search-icon"
+        src="./images/Search.svg"
+        alt="Search Icon"
+      >
+      <label for="inputBig">
+        <input
+          name="inputBig"
+          class="search-input search-input--big"
```

Comment:
```suggestion
          class="search-input"
```
better do not keep classes which not used